src/main/java/org/mule/extension/file/internal/command/LocalWriteCommand.java

```diff
@@ -62,8 +62,20 @@ public void write(String filePath, InputStream content, FileWriteMode mode,
    * {@inheritDoc}
    */
   @Override
-  public void write(String filePath, InputStream content, FileWriteMode mode,
-                    boolean lock, boolean createParentDirectory) {
+  public void write(String filePath, InputStream content, FileWriteMode mode, boolean lock, boolean createParentDirectory) {
+    commonWrite(filePath, content, mode, createParentDirectory, lock, NO_TIMEOUT);
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public void write(String filePath, InputStream content, FileWriteMode mode, boolean createParentDirectory, Long lockTimeout) {
+    commonWrite(filePath, content, mode, createParentDirectory, true, lockTimeout);
+  }
+
+  private void commonWrite(String filePath, InputStream content, FileWriteMode mode, boolean createParentDirectory, boolean lock,
```

Comment:
Same as above, why commonWrite? are there other ways of performing the write? I would lean towards doWrite.

Reply:
Done.